### ISSUE TITLE
Flow-control statements in FINALLY block rule does not raise an issue if FINALLY block, or flow-control statement, is inside the include

### ISSUE THREAD

**Issue by Ton-Pe** [Riverside-Software/sonar-openedge#1128]
Quite a minor issue, but wanted to report this. If the whole finally block, or flow-control statement, is inside an include file the error is not shown.

For example, this is not reported as an error:
test.p

```
DEFINE VARIABLE xError AS LOGICAL NO-UNDO.

{finally.i}
```

finally.i

```
FINALLY:
   IF xError THEN RETURN ERROR.
END FINALLY.
```

Neither is this:
test.p

```
DEFINE VARIABLE xError AS LOGICAL NO-UNDO.

FINALLY:
   {finally.i}
END FINALLY.
```

finally.i

```
   IF xError THEN RETURN ERROR.
```

Following code works as expected and error is reported:

```
DEFINE VARIABLE xError AS LOGICAL NO-UNDO.

FINALLY:
   IF xError THEN RETURN ERROR.
END FINALLY.
```

**Comment by Ton-Pe** [comment]
First problem, at least, is in both. Second example was tested only in Sonarlint.

OE plugin installed is 2.26.1. SonarQube version v9.9.5. OpenEdge 12.8 and SonarLint 9.3.3006
